src/wgpu_graphics_header.rs

```diff
@@ -487,50 +487,8 @@ pub async fn graphics_compile(
         // Use Triangles
         primitive: args.primitive_state,
 
-        /* wgpu::PrimitiveTopology::TriangleList,
-               color_states: match pipe_type {
-                   PipelineType::Stencil => &[],
-                   PipelineType::ColorWithStencil | PipelineType::Color => &[wgpu::ColorStateDescriptor {
-                       // Specify the size of the color data in the buffer
-                       // Bgra8UnormSrgb is specifically used since it is guaranteed to work on basically all browsers (32bit)
-                       format: wgpu::TextureFormat::Bgra8UnormSrgb,
-                       // Here is where you can do some fancy stuff for transitioning colors/brightness between frames. Replace defaults to taking all of the current frame and none of the next frame.
-                       // This can be changed by specifying the modifier for either of the values from src/dest frames or changing the operation used to combine them(instead of addition maybe Max/Min)
-                       color_blend: wgpu::BlendDescriptor::REPLACE,
-                       alpha_blend: wgpu::BlendDescriptor::REPLACE,
-                       // We can adjust the mask to only include certain colors if we want to
-                       write_mask: wgpu::ColorWrite::ALL,
-                   }],
```

Comment:
So is this?

Reply:
Some of these random snippets of wgpu code are defaults I originally had that are now up to the programmer. I've held on to them so that I know what the "correct" defaults were. These should probably become documentation instead as example inputs